browser/model/virtualbox.js

```diff
@@ -131,6 +132,9 @@ class VirtualBoxInstallWindows extends VirtualBoxInstall {
           return this.configure(installer);
         })
         .then((result) => {
+          if(result && this.targetDir) {
```

Comment:
Can I use this.getOption('install').location because getOption is not even declare any where.